.evergreen/evergreen.yml.in

```diff
@@ -1,4 +1,7 @@
 # Regenerate using `npm run update-evergreen-config`
+# https://jira.mongodb.org/browse/EVG-20276
+unset_function_vars: true
```

Comment:
I suspect that the e2e test failures on suse15 are because the `DISTRO_ID` addition in https://github.com/mongodb-js/mongosh/pull/1596 which triggers this evergreen bug, luckily they already have a workaround ready (which is supposed to be used for testing that evergreen configs have the correct behavior, but, well, let’s see how we do with this option set)

Reply:
Nope, all wrong. I think it’s actually coming from the fact that we switched the e2e test host from suse15 to suse15sp4 when we bumped to Node.js 20, and that mongodb-download-url does not understand that: https://github.com/mongodb-js/download-url/pull/264

This just went unnoticed because of the npm/Node.js incompatibility that was fixed in https://github.com/mongodb-js/mongosh/pull/1592.